ga4gh/client.py

```diff
@@ -28,6 +29,20 @@ def __init__(self, urlPrefix, debugLevel, workarounds, key):
         self._workarounds = workarounds
         self._key = key
 
+        # logging config
+        self.logger = logging.getLogger(__name__)
```

Comment:
Should logger be a public attribute or protected (i.e., `self._logger`)? I'd usually default to protected.